lib/app/index.js

```diff
@@ -44,7 +44,7 @@ const defaultTransition = <%=
 %>
 
 async function createApp (ssrContext) {
-  const router = createRouter(ssrContext)
+  const router = await createRouter(ssrContext)
```

Comment:
If the value of the expression following the await operator is not a Promise, it's converted to a resolved Promise.